order/interface.go

```diff
@@ -43,6 +44,11 @@ const (
 	// to use lease durations outside of the default/legacy 2016 block
 	// duration.
 	VersionLeaseDurationBuckets Version = 2
+
+	// VersionSelfChanBalance is the order version that added use of the
+	// self channel balance field. Only orders with this version are allowed
+	// to use the self channel balance field.
+	VersionSelfChanBalance Version = 3
```

Comment:
Is the plan to keep this version for sidecars as well, or have that be another version so 4 in this case?

Reply:
Also comment reminder to make this the default elsewhere in the codebase.

Reply:
The only place we currently actively set the value is in the CLI. I've updated the version there.